frontend/src/features/properties/map/acquisition/add/AddAcquisitionTeamForm.tsx

```diff
@@ -67,9 +69,21 @@ export const AddAcquisitionTeamForm: React.FunctionComponent<AddAcquisitionTeamF
                     variant="secondary"
                     className="px-2"
                     onClick={() => {
-                      arrayHelpers.remove(index);
+                      setShowRemoveTeamMemberModal(true);
                     }}
                   ></Button>
+                  <AcquisitionFormModal
+                    message="Are you sure you want to remove this row?"
+                    title="Remove Team Member"
+                    display={showRemoveTeamMemberModal}
+                    handleOk={() => {
+                      arrayHelpers.remove(index);
+                      setShowRemoveTeamMemberModal(false);
```

Comment:
This comes down to dev preference but I personally see low value in creating an extra class for a modal that is never going to be shared. That is, this could be refactored to just use the base **GenericModal** and pass "Ok" and "Cancel" as buttons text. Thoughts?

Reply:
This is the pattern used across application so followed the same. I was more inclined to use the GenericModal but did'nt want to deviate from existing norm. Please let me know your preference.